shared/src/main/java/me/neznamy/tab/shared/features/scoreboard/ScoreboardManagerImpl.java

```diff
@@ -35,15 +35,14 @@ public class ScoreboardManagerImpl extends TabFeature implements ScoreboardManag
 
     //defined scoreboards
     @Getter private final Map<String, me.neznamy.tab.api.scoreboard.Scoreboard> registeredScoreboards = new LinkedHashMap<>();
-    private me.neznamy.tab.api.scoreboard.Scoreboard[] definedScoreboards;
```

Comment:
Same as onlinePlayers array.

Reply:
that's right, I don't see the point in copying the array for any modification (toArray() copies the array)
forEach, in turn, does not copy anything

Reply:
forEach does create an iterator though.